lib/peeps.rb

```diff
@@ -0,0 +1,40 @@
+class Peep
+    attr_reader :user_id, :status, :id
+    require 'pg'
+
+    def initialize(user_id, status, id)
+       
+        @user_id = user_id
+        @status = status
+        @id = id
+
+    end
+
+    def self.all_peep
+
+        @peeps = []
+
+        rs = self.check_env.exec "SELECT * FROM chitter_table"
+
+    rs.map do |row|
+      
+      @peeps << Peep.new(row['user_id'], row['peep'], row['id'])
+    end
+        return @peeps
+    end
+
+    def self.post_peep(user_login, status)
+
+        self.check_env.exec "INSERT INTO chitter_table (user_id, peep) VALUES ('#{user_login}', '#{status}');"
+    end
+
+    private
+    def self.check_env
+      if ENV['ENVIRONMENT'] == 'test'
+        PG.connect :dbname => 'chitter_test'
+      else
+        PG.connect :dbname => 'chitter'
+      end
+    end
+
+end
```

Comment:
well named methods which made it clear what you were trying to implement. 

good use of using an object to create instances of the data from the tables - I did not do this. 

Something I noticed which I found a bit confusing was the use of status as the variable of the peep content, might want to consider using consistent naming structures from the datatable through to the model to make it clear  what data you are tyring to transmit - like with your use of user_id.